main/src/ca/uwaterloo/flix/api/lsp/provider/CompletionProvider.scala

```diff
@@ -239,10 +239,15 @@ object CompletionProvider {
     s"$name(${args.mkString(", ")})"
   }
 
+  private def getFilterTextForName(name: String): String = {
```

Comment:
Not to be pedantic, but I think we need a description.

Something about using only the name to avoid clash with symbols like [(.

Also, why do we have `(`?

Reply:
Please continue to be pedantic! Part of learning to contribute to any project is learning the project norms for this kind of thing.